### PR TITLE
Add instructions from uninstalling Redpanda and deleting K8s clusters

### PR DESCRIPTION
**Pull request by JakeSCahill**
Fixes https://github.com/redpanda-data/documentation-private/issues/1964

[Preview](https://deploy-preview-241--redpanda-docs-preview.netlify.app/current/deploy/deployment-option/self-hosted/kubernetes/local-guide/#uninstall-redpanda)

### CODE REVIEW

File: modules/deploy/partials/kubernetes/guides/uninstall.adoc [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+[tabs]
+======
+Helm + Operator::
++
+--
+[,bash]
+----
+kubectl delete -f redpanda-cluster.yaml --namespace <namespace>
+helm uninstall redpanda-controller --namespace <namespace>
+kubectl delete pod --all --namespace <namespace>
```
Comment (RafalKorepta) [inline modules/deploy/partials/kubernetes/guides/uninstall.adoc:14]: I'm not sure why we need to delete Pods. Did you saw Pods left behind?

**Review by RafalKorepta — APPROVED** [review]
LGTM

File: modules/deploy/partials/kubernetes/guides/uninstall.adoc (hunk 1 above)
Comment (JakeSCahill) [reply modules/deploy/partials/kubernetes/guides/uninstall.adoc:14]: The configurator Pods get left behind

**Review by micheleRP — APPROVED** [review]
lgtm

**Pull request merged**